src/components/ChatContent.tsx

```diff
@@ -0,0 +1,70 @@
+import {useRecoilValue } from 'recoil';
+import { ChatContentWrapper } from '../styles/style.chatcontent';
+import {roomList,userInfo} from '../store/atom';
+import ChatItem from './ChatItem';
+import {useRef,useEffect} from 'react';
+import {IChatRoom, IUser,IMessage,IRoomId} from '../store/interface';
+function ChatContent({roomid} : IRoomId): JSX.Element {
+  const roomLists = useRecoilValue<IChatRoom[]>(roomList);
+  const currentUser = useRecoilValue<IUser>(userInfo);
+  const chatList : IMessage[] = roomLists[roomid-1].messages;
+  const chatContent = useRef<HTMLDivElement>(null);
+  const prevTimeValue = useRef<String | null>(null);
+  const prevUserValue = useRef<Number | null>(null);
+  
+  const scrollChat = () => {
+    if (chatContent.current){
+      chatContent.current.scrollTop = chatContent.current.scrollHeight;
+    }
+  }
+  const makeTime = (time : number) =>{
+    const newTime = new Date(time);
+    let hours = newTime.getHours();
+    let minutes = newTime.getMinutes();
+    let hourFormat = hours < 12 ? `오전 ${hours}` : `오후 ${hours-12}`;
+    let minFormat = minutes < 10 ? `0${minutes}` : minutes.toString();
+    const madeTime = hourFormat + ":" + minFormat;
+    return madeTime;
+  }
+  useEffect(() => {
+    scrollChat();
+  },[roomLists]);
+  
+  useEffect(() => {
+    scrollChat();
+  },[currentUser]);
+  
+  return (
+    <ChatContentWrapper ref = {chatContent}>
+      {chatList.map((item,index) => {
+        let time = makeTime(item.id);
+        let showTime = true;
+        let showUser = false;

```

Comment:
```suggestion
        let [time, showTime, showUser] = [makeTime(item.id), true, false];
```
사소하지만... 변수만 주르륵 선언할 때 구조 분해 할당을 사용하면 좀 더 간추릴 수 있습니다 ㅎㅎ
취향차이니 참고만 하면 좋을 것 같아용